.gitignore

```diff
@@ -8,6 +8,9 @@ build_android_mips64/
 build_android_mips/
 build_android_x86/
 build_android_x86_64/
+Build/
```

Comment:
Let's remove these, doesn't seem like there's a need to keep adding to this file for every variation of junk people add in their directories.